Duanaire_Finn/PoemXLIII.trig

```diff
@@ -383,8 +381,7 @@
 <#Osgar>
     a foaf:Person;
     irishRel:nomName "Osgar";
-    rel:childOf <#Crochnaide>;
-    owl:sameAs <#Osgair>.
+    rel:childOf <#Crochnaide>.
```

Comment:
Do you have a particular reason for identifying this as someone other than Osgar, son of Oisín? Is another Osgar attested somewhere?